packages/terra-paginator/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ ChangeLog
 Unreleased
 ----------
 ### Changed
+* Updated wdio snapshots due to changes in button.
```

Comment:
Make sure in framework and clinical repo's there are no wdio failures. by referring this branch over there